packages/patternfly-4/react-core/src/components/LoginPage/LoginPage.js

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import { css } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+
+import Login from './Login';
+import LoginHeader from './LoginHeader';
+import LoginHeaderBrand from './LoginHeaderBrand';
+import LoginFooter from './LoginFooter';
+
+import LoginBox from './LoginBox';
+import LoginBoxHeader from './LoginBoxHeader';
+import LoginBoxBody from './LoginBoxBody';
+import LoginBoxFooter from './LoginBoxFooter';
+
+import { BackgroundImage } from '../BackgroundImage';
+import { Brand } from '../Brand';
+import { Text, TextContent } from '../Text';
+import { List } from '../List';
+
+const ListVariant = {
+  grid: 'grid',
+  inline: 'inline'
+};
+
+const propTypes = {
+  /** Anything that can be rendered inside of the LoginPage */
```

Comment:
I wonder if there are any components here that are always required  for a loginPage.
@christiemolloy do you know?

Reply:
looping in @andybraren and @mcarrano to take a look at this

Reply:
I believe all of these would be required for any generic LoginPage component, except maybe “Brand” (if that’s a PF-brand-specific thing).

Reply:
I think for ease of use we should not require some of these, for example a consumer might not need a language dropdown. I think the only required piece really should be the login form when it gets added later and maybe the loginTitle

Reply:
I agree, I think that makes sense @jschuler  also certain components can't be required at the moment for example like loginFooterContent since we don't have the social icons component yet. Might be a bit much to require everything for this.

Reply:
I think the loginTitle and login form should be required when that gets added as well as the brandImgAlt and backgroundImgAlt since we want them to provide alt text.